boa_cli/src/main.rs

```diff
@@ -184,6 +224,31 @@ where
     Ok(())
 }
 
+fn generate_flowgraph(
+    context: &mut Context,
+    src: &[u8],
+    flowgraph: FlowgraphFormat,
```

Comment:
I would rename this to `format`, and the `flowgraph_direction` to `direction`, just to avoid repetition and make it more clear.